spec/javascripts/title_module_spec.js

```diff
@@ -0,0 +1,26 @@
+var editor; var module;
+
+describe("TitleModule", function() {
+  beforeAll(function() {
+    fixture = loadFixtures('index.html');
+
+    editor = new PL.Editor({
+      textarea: $('.ple-textarea')[0]
+    });
+
+    module = new PL.TitleModule(editor, {});
+  });
+
+  it("value, valid, key", function() {
+    expect(module).not.toBeUndefined();
+    expect(module.key).toBe('title');
+
+    expect(module.value()).not.toBe(false);
+    expect(module.value()).toBe('');
+
+    module.value('My new post:');
```

Comment:
Here, I'm wondering if we should test specifically the idea of adding/removing items, and confirming that they appear/disappear in the right places? See how @Shulammite-Aso has used Jest to create a set of conditions and then is testing whether the outcome is correct, here? 

https://github.com/jywarren/woofmark/pull/15/files#diff-635478b67938477db9d8a9bc88585289R10

Thank you!

Reply:
Will do it soon thanks.

Reply:
Hi Nitin, are you able to try typing one of the terms in the autocomplete mock JSON that's already in the repo, and then confirming that we can add a related item?